src/components/movie-view/movie-view.jsx

```diff
@@ -1,27 +1,45 @@
+import "./movie-view.scss";
+
 export const MovieView = ({ movie, onBackClick }) => {
   return (
     <div className="movie-view">
       <div className="movie-title">
         <h1>{movie.title}</h1>
       </div>
       <div className="movie-description">
-        <p><strong>Description: </strong>{movie.description}</p>
+        <p>
+          <strong>Description: </strong>
+          {movie.description}
+        </p>
       </div>
       <div className="movie-genre">
-        <p><strong>Genre:</strong> {movie.genre.name}</p>
+        <p>
+          <strong>Genre:</strong> {movie.genre.name}
+        </p>
       </div>
       <div className="movie-director">
-        <p><strong>Director:</strong> {movie.director.name}</p>
+        <p>
+          <strong>Director:</strong> {movie.director.name}
+        </p>
       </div>
       <div className="movie-rating">
-        <p><strong>Rating:</strong> {movie.rating.$numberDouble}</p>
+        <p>
+          <strong>Rating:</strong> {movie.rating.$numberDouble}
+        </p>
       </div>
       <div className="movie-image">
         <img src={movie.imageURL} alt={movie.title} />
       </div>
+     
       <div className="movie-buttons">
-        <button onClick={onBackClick}>Back</button>
+        <button
+          onClick={onBackClick}
+          className="back-button"
+          style={{ cursor: "pointer" }} //double curly braces indicate that the value is a JavaScript object
```

Comment:
It's always best practice to use external style like .scss file to add your styling. Inline style is limited to situations where you can't avoid it(especially if you need to override default inherited styles from library).